### ISSUE TITLE
Request to remove invalid CSS property

### ISSUE THREAD

**Issue by keefyhub** [OwlCarousel2/OwlCarousel2#2256]
I ran one of my sites through a CSS validator and found an invalid CSS property in owl carousel core

```
.owl-carousel .owl-dot,
.owl-carousel .owl-nav .owl-next,
.owl-carousel .owl-nav .owl-prev {
  cursor: hand;
  ...
}
```

[validator link](https://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fowlcarousel2.github.io%2FOwlCarousel2%2Fdemos%2Fbasic.html&profile=css3svg&usermedium=all&warning=1&vextwarning=&lang=en)

Is there any need for this or can I put in a PR to remove?

*There are also issues in theme.css but as that is not required I have not included it

**Comment by pascalporedda** [comment]
Feel free to create a PR, instead of removing you might simply change it to cursor: pointer;

Thank you. Which errors did you found in the theme.css?

**Comment by keefyhub** [comment]
@pascalporedda there is both `pointer` and `hand` in the CSS so I will create a PR to remove the invalid one.

For the theme, there are 2 properties that are flagged as invalid

```
.top-bar .toggle-topbar.menu-icon a span:hover:after | "" is not a box-shadow value : 0 0 0 1px "",0 7px 0 1px "",0 14px 0 1px ""
.top-bar-section | 0 is not a transition value : none 0 0
```
I guess this could be fixed by replacing the `""` with `0` (although I am unsure of the implications) and changing `none 0 0` to just `none` (Im not sure if you have to zero the duration or delay when setting to none.

https://jigsaw.w3.org/css-validator/validator?uri=https%3A%2F%2Fowlcarousel2.github.io%2FOwlCarousel2%2Fdemos%2Fbasic.html&profile=css3svg&usermedium=all&warning=1&vextwarning=&lang=en

**Comment by pascalporedda** [comment]
Yeah they are simply invalid, it should be fine to replace these values with the correct one. I've added this to my todo list for this project. Lot of stuff to do here..

I've merged your PR and it is contained in version 2.3.3
